components/contribution-flow/StepSummary.js

```diff
@@ -245,7 +245,7 @@ const StepSummary = ({
   const tierType = tier?.type;
   const hostCountry = get(collective.host, 'location.country');
   const collectiveCountry = collective.location.country || get(collective.parent, 'location.country');
-  const currency = tier?.amount.currency || collective.currency;
+  const currency = isCrypto ? stepDetails.currency.value : tier?.amount.currency || collective.currency;
```

Comment:
linter feedback

Reply:
```suggestion
  const currency = isCrypto ? stepDetails.currency.value : (tier?.amount.currency || collective.currency);
```

Easier to read with parenthesis?

Reply:
Thanks. Corrected that. 👍🏽

Reply:
Done. 👍🏽

Reply:
@znarf : Actually this one isn't possible as prettier seem to think that not having parenthesis is better. So either we have to make a prettier exception here or just leave it without parenthesis.